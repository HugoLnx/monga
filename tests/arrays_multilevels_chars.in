void main() {
	char[][] array;
	array = new char[][2];
	array[0] = new char[2];
	array[1] = new char[3];
  array[0][0] = 256;
  array[0][1] = 65;
  array[1][0] = 66;
  array[1][1] = 67;
  array[1][2] = 68;
	printf("array[0][0]: %d\n", array[0][0]);
	printf("array[0][1]: %d\n", array[0][1]);
	printf("array[1][0]: %d\n", array[1][0]);
	printf("array[1][1]: %d\n", array[1][1]);
	printf("array[1][2]: %d\n", array[1][2]);
}
