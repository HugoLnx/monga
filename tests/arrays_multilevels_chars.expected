array[0][0]: 0
array[0][1]: 65
array[1][0]: 66
array[1][1]: 67
array[1][2]: 68
